src/server/templates/settings.html

```diff
@@ -2260,6 +2266,15 @@ <h2>{{ __('Alert') }}</h2>
 			</div>
 		</div>
 
+		<div id="restart_confirm" class="priority-message-interrupt mfp-hide popup">
+			<h2>Alert</h2>
+			<div class="popup-content">
+				<p>{{ __('Restart server process?') }}</p>
```

Comment:
I think it should be "Restart RotorHazard server?", to match other verbiage.

Reply:
Should we change "Shut down server" to "Shut down server hardware" (and same for reboot), to differentiate from server process?